### DISCUSSION TITLE
format doesn't works

### DISCUSSION THREAD

**Discussion by temuccio** [js-fns/date-fns#2048] [Q&A]
Hi all, 
I have interested to use date-fns in my react-native project.
I use `react-native version 0.62.2` and `date-fns version 2.16.1`.
after the import format such as
```import {format} from 'date-fns';```
I have try this example:
```
console.log('pre date');
var result = format(new Date(2014, 1, 11),'MM/DD/YYYY');
console.log.og(result);
console.log('after date');
```
The console print "pre date" but after the code is blocking (without error).
I have try also 
```
var result = format(new Date(),'MM/DD/YYYY');
```
but I have the same problem.
Where I wrong?

Thanks

**Comment by tkrebs2** [comment]
I did a quick copy and paste and was getting an error with your `format()` method, specifically the format argument.
Try `var result = format(new Date(2014, 1, 11), 'MM/dd/yyyy');`
